examples/nlp/text_classification/conf/config.yaml

```diff
@@ -0,0 +1,71 @@
+defaults:
+  - trainer: pl
+  - model: text_classification_with_bert
+
+trainer:
+  fast_dev_run: true
+
```

Comment:
please set fast_dev_run to false. If the yaml files are so small now can we create a folder under examples/nlp/config and store configs for all models there?

Reply:
oh wait, you have another yaml under models, why do we need 2 files?